_includes/share-article-list.html

```diff
@@ -0,0 +1,25 @@
+{% assign base_url = 'https://womencodingcommunity.com' %}
+{% assign post_url = page.url %}
+
+{% assign share_list = site.data.blog-share %}
+
+<ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdown-share-article">
+    {% for entry in share_list %}
```

Comment:
you don't need to create a for with a list, you can use directly if/else (check footer for reference)

Reply:
Yeah, saw that but initially didn't want to use that approach because it meant more duplicating more code i.e 

```
<li class="d-flex align-items-center">
            <a class="dropdown-item d-flex align-items-center" href="{{entry.prefill-url}}{{base_url}}{{post_url}}" target="_blank">
                <span class="share-icon">
```